hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystemWithFSO.java

```diff
@@ -85,6 +85,12 @@ public TestOzoneFileSystemWithFSO(boolean setDefaultFs,
   @Override
   public void cleanup() {
     super.cleanup();
+    try {
+      deleteRootDir();
+    } catch (IOException e) {
+      LOG.info("Failed to cleanup DB tables.", e);
+      fail("Failed to cleanup DB tables." + e.getMessage());
+    }
```

Comment:
This recently added part seems unrelated.

Reply:
> This recently added part seems unrelated.

Ok, its handled. I think it came while resolving conflict. Checked carefully.